SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -257,13 +264,13 @@ QueryConstraint InterestFactory::CreateSystemDefinedConstraints() const
 	return SystemDefinedConstraints;
 }
 
-QueryConstraint InterestFactory::CreateCheckoutRadiusConstraints() const
+QueryConstraint InterestFactory::CreateCheckoutRadiusConstraints(const InterestRequest& Request) const
 {
 	// If the actor has a component to specify interest and that indicates that we shouldn't generate
 	// constraints based on NetCullDistanceSquared, abort. There is a check elsewhere to ensure that
```

Comment:
Total nit, as it is unrelated to your change, but this "check elsewhere" is not enforced in any way. Do you think this should be a precondition?

Reply:
We can do, although I do not have much context here. I have no idea where it would be added.

Reply:
yeah, no worries, it's not all that important (and still unrelated). it would be an if here that says if more than one, panic in some way with a hopefully helpful error message.

Reply:
Oh sure I can add that xD

Reply:
I'm not sure I follow - there's code in `InterestFactory::AddUserDefinedQueries` to error if there's something wrong with the number of UActorInterestComponents

Reply:
This code makes an assumption here that it is in the correct state because of that other check, but if that other check was changed then it wouldn't be immediately obvious that this precondition wouldn't hold any more. It is probably fine leaving it as is, as multiple actor interest components will just result in undefined behaviour, as the function isn't expecting it